compose/mpp/demo/src/uikitMain/kotlin/androidx/compose/mpp/demo/main.uikit.kt

```diff
@@ -2,9 +2,54 @@
 package androidx.compose.mpp.demo
 
 import ApplicationLayoutExamples
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Button
+import androidx.compose.material.Text
 import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.interop.LocalUIViewController
 import androidx.compose.ui.main.defaultUIKitMain
 import androidx.compose.ui.window.ComposeUIViewController
+import platform.UIKit.UINavigationController
+import platform.UIKit.childViewControllers
+import platform.UIKit.navigationController
+
+@Composable
+fun NativePopupPresenter() {
```

Comment:
It shouldn't be placed in main file

Reply:
Moved around. I will resolve merge conflict, once your branch is ready. Should be trivial.